lib/react.dart

```diff
@@ -192,15 +190,13 @@ abstract class Component {
       'Only supported in the deprecated Component, and not Component2. Use createRef or a ref callback instead.')
   set ref(RefMethod value) => _ref = value;
 
-  dynamic _jsRedraw;
+  late Function _jsRedraw;
```

Comment:
Is `Function` as specific as we can be?

I know dynamic call-sites are no bueno for dart2js compile times - so its great that we're getting rid of that here... but are functions with no known arg arity something to avoid if we can help it?

Reply:
Good catch, yeah I can definitely make this more specific as `void Function()`

Reply:
Addressed in https://github.com/Workiva/react-dart/pull/373/commits/ee90271ade553281b32779002dbb03325e20db4d